src/tests/conformance/post_handshake/query/get_block.rs

```diff
@@ -101,10 +101,124 @@ async fn c010_UNI_ENS_BLOCK_REQ_get_block() {
         });
         assert!(synthetic_node.unicast(net_addr, message).is_ok());
 
-        // Expect a UniEnsBlockRsp response with a block with the same round.
+        // Expect a UniEnsBlockRsp response with a block with the same round and also a certificate.
         let check = |m: &Payload| {
             matches!(&m, Payload::TopicMsgResp(TopicMsgResp::UniEnsBlockRsp(rsp))
-                     if rsp.block.is_some() && rsp.block.as_ref().unwrap().round == round)
+                     if rsp.block.is_some() && rsp.block.as_ref().unwrap().round == round && rsp.cert.is_some())
+        };
+        assert!(
+            synthetic_node.expect_message(&check).await,
+            "the UniEnsBlockRsp response is missing"
+        );
+    }
+
+    // Gracefully shut down the nodes.
+    synthetic_node.shut_down().await;
+    node.stop().expect("unable to stop the node");
+}
+
```

Comment:
I tried to write a generic `send_UNI_ENS_BLOCK_REQ_and_then_expect` function, but then I needed to provide a round as an argument, which would mean we would be starting and stopping synth nodes for each round which is too much.

So I gave up on that idea because I don't mind little code duplication as long as tests are readable.